docs/code_examples/parallel_rps.py

```diff
@@ -128,7 +130,8 @@ def reset(self, seed=None, options=None):
         """
         self.agents = self.possible_agents[:]
         self.num_moves = 0
-        observations = {agent: NONE for agent in self.agents}
+        # the observations should be numpy arrays even if there is only one value
+        observations = {agent: np.array(NONE) for agent in self.agents}
```

Comment:
And honestly this code is pretty old and should probably be totally redone, doing np.array(NONE) is super sloppy and confusing in itself as well.

Reply:
Yeah, the NONE was a confusing for me. I misread it as None and couldn't figure out why it wasn't working as expected

Reply:
Maybe as a quick, slightly less confusing option, change NONE to NO_MOVE? That doesn't fix the root problem, but it avoids overlap with the None keyword

Reply:
Ohhh okay I re-read the code and I see it's just NO_MOVE = 3, okay that's clear and concise enough IMO. With parallel envs you can sometimes have agents do no move, so you need to handle that.